tests/benchmark_test.py

```diff
@@ -222,3 +232,9 @@ def test_functions_call_shape(fun):
     x_list[0] = numpy.linspace(fun.lower_bounds,mid_point,50)
     x_list[1] = numpy.linspace(mid_point,fun.upper_bounds,50)
     assert fun(x_list).shape == (2,50)
+
+@pytest.mark.parametrize("fun",functions)
+def test_functions_optimum(fun):
+    """Test all functions optimums are at their locations"""
+    print(fun.name)
```

Comment:
```suggestion
```

Reply:
Whenever this function errors or fails, it doesn't print out the name of the function, just the order in the list. I added a print so it's easier to determine what function needed to be investigated.